python/pyspark/mllib/tests.py

```diff
@@ -25,7 +25,8 @@
 import array as pyarray
 from time import time, sleep
 
-from numpy import array, array_equal, zeros, inf, all, random
+from numpy import (
+    array, array_equal, zeros, inf, random, exp, dot, all, mean)
```

Comment:
Sure, I thought this was some sort of micro-optimization and followed the convention that is already there.